web/pgadmin/tools/sqleditor/static/js/components/QueryToolDataGrid/index.jsx

```diff
@@ -116,7 +116,7 @@ function CustomRow(props) {
   };
   return (
     <RowInfoContext.Provider value={rowInfoValue}>
-      <Row ref={rowRef} onKeyDown={handleKeyDown} {...props} />
+      <Row ref={rowRef} onKeyDown={handleKeyDown} {...props} rangeSelectionMode={true}/>
```

Comment:
there is no such prop rangeSelectionMode on Row component.

Reply:
fixed, thanks